src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -35,6 +35,7 @@ class TextDescriptor extends Descriptor
     protected function describeRouteCollection(RouteCollection $routes, array $options = array())
     {
         $showControllers = isset($options['show_controllers']) && $options['show_controllers'];
+        $filter = isset($options['filter']) ? $options['filter'] : '';
```

Comment:
To avoid issues with the regular expression used later, this should be:

```php
$filter = isset($options['filter']) ? preg_quote($options['filter'], '/') : '';
```